### ISSUE TITLE
validation loss increse and validation acc is decrease in my rnn model.

### ISSUE THREAD

**Issue by ujas3279** [keras-team/keras#13942]
I am working on abusive and violent content detection. when i train my model than its shows like that.

    Train on 9087 samples, validate on 2125 samples
    Epoch 1/5
    9087/9087 [==============================] - 33s 4ms/step - loss: 0.3193 -accuracy: 0.8603 -         val_loss: 0.2314 - val_accuracy: 0.9322
    Epoch 2/5
    9087/9087 [==============================] - 33s 4ms/step - loss: 0.1787 - accuracy: 0.9440 - val_loss: 0.2039 - val_accuracy: 0.9356
    Epoch 3/5
    9087/9087 [==============================] - 32s 4ms/step - loss: 0.1148 - accuracy: 0.9637 - val_loss: 0.2569 - val_accuracy: 0.9180
    Epoch 4/5
    9087/9087 [==============================] - 33s 4ms/step - loss: 0.0805 - accuracy: 0.9738 - val_loss: 0.3409 - val_accuracy: 0.9047
    Epoch 5/5
    9087/9087 [==============================] - 36s 4ms/step - loss: 0.0599 - accuracy: 0.9795 - val_loss: 0.3661 - val_accuracy: 0.9082

here i use following layers in my model-

    model = Sequential()
    model.add(Embedding(8941, 256,input_length=20))
    model.add(LSTM(32, dropout=0.1, recurrent_dropout=0.1))
    model.add(Dense(32,activation='relu'))
    model.add(Dropout(0.4))
    model.add(Dense(4, activation='sigmoid'))
    model.summary()

    model.compile(loss='binary_crossentropy',
              optimizer=optimizers.Adam(lr=0.001),
              metrics=['accuracy'])

    history=model.fit(x, x_test,
          batch_size=batch_size,
          epochs=5,
          verbose=1,
          validation_data=(y, y_test))

please help me to solve this problem........

**Comment by sushreebarsa** [comment]
@ujas3279  Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
